changelog.d/11200.bugfix

```diff
@@ -0,0 +1 @@
+Fixes a long-standing bug wherein a missing Content-Type header when downloading remote media causes Synapse to throw an error.
```

Comment:
```suggestion
Fix a long-standing bug wherein a missing `Content-Type` header when downloading remote media would cause Synapse to throw an error.
```